### ISSUE TITLE
Changed ubuntu server version number to make it work with packer build

### ISSUE THREAD

**Issue by bradib0y** [udacity/devops-intro-project#71]
In application-server.json,

I changed ubuntu version number to make it work. The original was this:
"PACKER_BOX_NAME": "ubuntu-14.04.4-server-amd64",

I changed version number ending to 5, like this:
"PACKER_BOX_NAME": "ubuntu-14.04.5-server-amd64",

**Comment by bradib0y** [comment]
And now the checksum fails, it's some sort of hash which is probably supposed to detect whether it is the exact same version or not... :(

**Comment by bradib0y** [comment]
I resolved that with checksum:
"iso_checksum": "dde07d37647a1d2d9247e33f14e91acb10445a97578384896b4e1d985f754cc1"

After that, I had problem with "VBoxManage" and "VT-x".

I was thinking that the VirtualBox is maybe missing from the environment path variables, so I added it. It didn't resolve the issue though. 

Then on forums I found that with "VT-x", the Windows Hyper-V service might cause problems, so I disabled it in Control Panel/Programs and Features/Windows services/Hyper-V

After all this, the scipts started working and installing the virtual machine by itself, completely autonomously.
